docs/FAQ.md

```diff
@@ -66,6 +66,58 @@ One session of Zulip Terminal represents a connection of one user to one Zulip s
 
 Since we expect the above to be straightforward for most users and it allows the code to remain dramatically simpler, we are unlikely to support multiple Zulip servers within the same session in at least the short/medium term. However, we are certainly likely to move towards a system to make access of the different servers simpler, which should be made easier through work such as [zulip-terminal#678](https://github.com/zulip/zulip-terminal/issues/678).
 
+## How do I use autocompletes in ZT?
+
+Composing messages could be a tiring task when you have to type all things by yourself. Autocompletes come to rescue here! ZT currently supports some basic autocompletes, for different compose boxes.
+
+### Hotkeys triggering autocomplete
+
+In most places, hitting `tab` allows toggling autocompletes. In ZT, however, `ctrl f` and `ctrl r` are used for toggling purpose:
```

Comment:
Re shitf+tab, could you elaborate our use-case for it? I tried some things but wasn't able to figure out its working.

Reply:
We don't currently use `shift tab`, but in my past experience it's used for cycling backwards, ie. opposite of `tab`.

Originally I thought we could mention it since if there was a reason we didn't use `shift tab` then that may help explain our current key setup in this kind of document. Unfortunately not :(